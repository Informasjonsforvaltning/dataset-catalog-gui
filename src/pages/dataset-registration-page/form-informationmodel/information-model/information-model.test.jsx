import React from 'react';
import { shallow } from 'enzyme';
import InformationModel from './information-model.component';

test('should render InformationModel correctly', () => {
  expect(shallow(<InformationModel />)).toMatchSnapshot();
});
