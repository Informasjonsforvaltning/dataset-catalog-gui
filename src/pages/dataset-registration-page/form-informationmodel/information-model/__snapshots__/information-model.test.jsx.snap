// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render InformationModel correctly 1`] = `""`;
